grpc.go

```diff
@@ -54,8 +54,10 @@ const (
 )
 
 var (
-	sysfsCPUOnlinePath = "/sys/devices/system/cpu"
-	sysfsMemOnlinePath = "/sys/devices/system/memory"
+	sysfsCPUOnlinePath     = "/sys/devices/system/cpu"
+	sysfsMemOnlinePath     = "/sys/devices/system/memory"
+	sysfsConnectedCPUsPath = filepath.Join(sysfsCPUOnlinePath, "online")
+	sysfsDockerCpusetPath  = "/sys/fs/cgroup/cpuset/docker/cpuset.cpus"
```

Comment:
dump question why 'docker'?

Reply:
because we are using `libcontainer` and by default it creates the cgroup with that name

Reply:
would be nice to have it in a comment, but thanks for clarify.

Reply:
@devimc if libcontainer creates this, is there any way we could retrieve this from libcontainer itself instead of hardcoding it ?

Reply:
I didn't find a way to get this from `libcontainer`